index.html

```diff
@@ -460,6 +473,23 @@ <h2>
             </li>
           </ol>
         </dd>
+        <dt>
+          <dfn>touches</dfn>
+        </dt>
+        <dd>
+          <p>
+            A [=list=] of {{GamepadTouch}} objects generated from all touch
+            surfaces.
+          </p>
+          <p>
+            The {{Gamepad/touches}} getter steps are:
+          </p>
+          <ol>
+            <li>Return the [=list=] of the {{Gamepad/[[touches]]}} the internal
```

Comment:
nit: maybe missing an "in"? "...{{Gamepad/[[touches]]}} _in_ the internal slot"

Reply:
I think this can be:

Return [=this=].{{Gamepad/[[touches]]}}.